src/Tests/Microsoft.NET.Sdk.Razor.Tests/TestFiles/Generated/Counter.razor.rz.scp.css

```diff
@@ -0,0 +1,3 @@
+.counter {
```

Comment:
Are these used by tests?

Reply:
Yes, they are used by the concatenate file tests.